finesse/logger.py

```diff
@@ -13,9 +14,15 @@ def initialise_logging() -> None:
     log_path.mkdir(parents=True, exist_ok=True)
     filename = log_path / f"{datetime.now().strftime('%Y%m%d_%H-%M-%S')}.log"
 
+    # Allow user to set log level with environment variable
+    log_level = os.environ.get("FINESSE_LOG_LEVEL") or "INFO"
+    log_level = log_level.upper()
```

Comment:
```suggestion
    log_level = (os.environ.get("FINESSE_LOG_LEVEL") or "INFO").upper()
```
I've never been a big fan of assigning a variable just to change immediately afterwards.